packages/react-components/react-aria/src/activedescendant/scrollIntoView.ts

```diff
@@ -1,11 +1,8 @@
-export const scrollIntoView = (target: HTMLElement | null | undefined, winArg?: Window | null) => {
+export const scrollIntoView = (target: HTMLElement | null | undefined) => {
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31470&commitId=ab70d37b312e60327cc7fdc4bde373c8e7117526&env=prod&clientType=FLUENTUI)


  <details><summary><b>SwatchPicker Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SwatchPicker Converged.Shape.default.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>